.eslintrc.yml

```diff
@@ -88,3 +91,16 @@ overrides:
       - test/validStrings.mjs
     rules:
       no-console: off
+  - files:
+      - test/*.mjs
+      - test/**/*
+      # TODO Fix these files!
+      - lib/calendar.ts
```

Comment:
I've only been working on ecmascript.ts (with a few minor changes to calendar.ts that would be easy to rebase on top of any work you do), so all other files are fair game for this PR. No need to limit to just Plain types.

Reply:
Done.